doc/parameter.rst

```diff
@@ -225,6 +225,13 @@ Parameters for Tree Booster
     list is a group of indices of features that are allowed to interact with each other.
     See tutorial for more information
 
+Additional parameters for `hist` tree method
+================================================
+
+* ``single_precision_histogram``, [default=``false``]
+
+  - The experimental parameter single_precision_histogram can be set to True to enable building histograms using single precision.
```

Comment:
Also, I prefer to use similar to GPU description and don't name it as experimental feature. For example:
```Use single precision to build histograms instead of double precision.```

Reply:
description was changed.

Reply:
description was changed.